### ISSUE TITLE
 make uninstall runs rm -rf /usr/local/include

### ISSUE THREAD

**Issue by xuchunyang** [janet-lang/janet#129]
After `make uninstall`, my whole `/usr/local/include` is gone.

```
~/src/janet $ make --dry-run uninstall
rm /usr/local/bin/../build/janet
rm -rf /usr/local/include
~/src/janet $
```

**Comment by bakpakin** [comment]
Sorry about this, I was on vacation or I would have fixed this bug sooner. In the meantime, the Meson build is more standard and should have a working uninstall target. We may even transition to Meson fully. I will fix the Makefile very soon and push a fix.

**Comment by bakpakin** [comment]
Should be fixed as of f7f2f5e84ff19c802af28c09aca5177c8dc6fa68
